### PR TITLE
Сreate stars to ag grid for feedbacks

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- add feedbacks and names of users
- add component for stars for ag-grid
Снимок экрана 2021-12-03 в 1 50 36 PM
https://trello.com/c/cTyhpOHs/182-fe-add-stars-to-ag-grid-for-feedbacks-hr-interviewer

### CODE REVIEW

File: src/components/calendar/Calendar.jsx [hunk 1]
```diff
@@ -17,7 +17,11 @@ dayjs.extend(advancedFormat);
 
 function splitInterval(start, end, step) {
   const result = [];
-  for (let ts = Number(dayjs(start).format('X')); ts < Number(dayjs(end).format('X')); ts+=step) {
+  for (
+    let ts = Number(dayjs(start).format('X'));
```
Comment (nsukhotskaya) [inline src/components/calendar/Calendar.jsx:21]:
```suggestion
    let ts = +(dayjs(start).format('X'));
```

File: src/components/calendar/Calendar.jsx [hunk 2]
```diff
@@ -17,7 +17,11 @@ dayjs.extend(advancedFormat);
...
+    ts < Number(dayjs(end).format('X'));
```
Comment (nsukhotskaya) [inline src/components/calendar/Calendar.jsx:22]:
```suggestion
    ts < +(dayjs(end).format('X'));
```

File: src/components/starFormatter/StarFormatter.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Rating, Box } from '@mui/material';
+
+const StarFormatter = ({ value }) =>
+  value === undefined ? (
```
Comment (nsukhotskaya) [inline src/components/starFormatter/StarFormatter.jsx:5]: What about `!value ? smth1: smth2` ?

File: src/components/starFormatter/StarFormatter.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    <Box>{value}</Box>
```
Comment (nsukhotskaya) [inline src/components/starFormatter/StarFormatter.jsx:6]: Stop. Do you show  user `undefined ` text ????

File: src/components/starFormatter/StarFormatter.jsx (hunk 3 above)
Comment (nsukhotskaya) [inline src/components/starFormatter/StarFormatter.jsx:5]: Trye to use `&&` instead of '?'

File: src/constants/reformatCandidates.js [hunk 5]
```diff
@@ -7,16 +7,17 @@ export const reformatCandidates = (candidates) =>
...
+      candidate.feedbacks &&
+      candidate.feedbacks.map((item) => item.finalEvaluation);
+    const [hrName, interviewerName] = users;
+    newObj.hr = hrName;
+    newObj.interviewer = interviewerName;
+    const [hrFeedback, interviewerFeedback] = feedbacks;
+    newObj.hrReview = hrFeedback;
+    newObj.interviewerReview = interviewerFeedback;
+
     return newObj;
```
Comment (itsnotpolza) [inline src/constants/reformatCandidates.js:22]:
```suggestion
    const users =
      candidate.users && candidate.users.map((item) => item.userName);
    const feedbacks =
      candidate.feedbacks &&
      candidate.feedbacks.map((item) => item.finalEvaluation);
    const [hrName, interviewerName] = users;
    newObj.interviewer = interviewerName;
    const [hrFeedback, interviewerFeedback] = feedbacks;
    newObj.hrReview = hrFeedback;
    newObj.interviewerReview = interviewerFeedback;

    return {hr: hrName, interviewer: interviewerName, ...};
```

Comment (itsnotpolza) [reply src/constants/reformatCandidates.js:22]: this is a very rough suggestion there's no need to create a separate object

**Review by nsukhotskaya — APPROVED** [review]

**Review by itsnotpolza — APPROVED** [review]

**Pull request merged**